src/System.ComponentModel.Annotations/ref/System.ComponentModel.Annotations.csproj

```diff
@@ -12,6 +12,10 @@
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsNetFx)' == 'true'">
     <Reference Include="mscorlib" />
+    <Reference Include="System" />
```

Comment:
@danmosemsft I addressed the [code review](https://github.com/dotnet/corefx/pull/33312#pullrequestreview-174151276)